pkg/operators/limiter/limiter_test.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package limiter is a data operator that limits the number of entries in each
+// batch of data. This operator is only enabled for data sources of type array.
+// A great scenario for this operator is when you are already sorting data
+// within an array of data and you want to filter out the top `X` entries.
+package limiter
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/datasource"
+	"github.com/stretchr/testify/require"
```

Comment:
The `require` import should be in its own block above the `inspektor-gadget` import